tests/testsuite/cmd_duration.rs

```diff
@@ -0,0 +1,83 @@
+use ansi_term::Color;
+use std::fs;
+use std::io;
+use std::path::Path;
+use tempfile::TempDir;
+
+use crate::common::{self, TestCommand};
+
+#[test]
+fn config_blank_duration_1s() -> io::Result<()> {
+    let output = common::render_module("cmd_duration")
+        .arg("--cmd-duration=1")
+        .output()?;
+    let actual = String::from_utf8(output.stdout).unwrap();
+
+    let expected = "";
+    assert_eq!(expected, actual);
+    Ok(())
+}
+
+#[test]
+fn config_blank_duration_5s() -> io::Result<()> {
+    let output = common::render_module("cmd_duration")
+        .arg("--cmd-duration=5")
+        .output()?;
+    let actual = String::from_utf8(output.stdout).unwrap();
+
+    let expected = format!("{} ", Color::Yellow.bold().paint("took 5s"));
+    assert_eq!(expected, actual);
+    Ok(())
+}
+
+#[test]
+#[ignore]
```

Comment:
Because the config files that are generated are tempfiles created in a temp directory, there's no harm with running these tests on all machines. 🙂

```suggestion
```

Reply:
So the `#[ignore]` is usually used for tests that might (significantly) alter the test computer's state?

Reply:
Yep. Ignored tests are run in Docker with `./integration_test`, but are not run with `cargo test`.